res/translations/en.json

```diff
@@ -210,6 +210,7 @@
     "Show notification on track change": "Show notification on track change",
     "Show play queue": "Show play queue",
     "Show year in album grid cards": "Show year in album grid cards",
+    "Show year in with album name": "Show year in with album name",
```

Comment:
This new string is slightly confusing to me. What about something like "Show year in album and now playing cards"?